src/styles/blocks/variables.scss

```diff
@@ -0,0 +1,4 @@
+$blue-hover: #00acdc;
+$grey-star-color: #f3f3f3;
+$standart-inline-pad: 16px;
```

Comment:
The variable name '$standart-inline-pad' contains a typo. It should be '$standard-inline-pad' to ensure correct spelling and consistency across your styles.